src/utils/domUtils.ts

```diff
@@ -9,99 +9,58 @@ export module Utils {
      * @param {d3.Selection} element
      * @returns {SVGRed} The bounding box.
      */
-    export function getBBox(element: d3.Selection<any>): SVGRect {
+    export function getBBox(element: d3.Selection<any>) {
       var bbox: SVGRect;
       // HACKHACK: Firefox won't correctly measure nodes with style "display: none" or their descendents (FF Bug 612118).
       try {
         bbox = (<any> element.node()).getBBox();
       } catch (err) {
-        bbox = {
-          x: 0, y: 0, width: 0, height: 0
-        };
+        bbox = { x: 0, y: 0, width: 0, height: 0 };
       }
       return bbox;
     }
 
-    export var POLYFILL_TIMEOUT_MSEC = 1000 / 60; // 60 fps
-    export function requestAnimationFramePolyfill(fn: () => any): void {
+    export var POLYFILL_TIMEOUT_MILLISECONDS = 1000 / 60; // 60 fps
+
+    export function requestAnimationFramePolyfill(fn: () => any) {
```

Comment:
Now that we decided to expose these things.

Do you think `requestAnimationFramePolyfill` is a good name?